EXT_SpeciesType.md

```diff
@@ -0,0 +1,98 @@
+Convention for Specifying Particle Species 
+==========================================
+
+openPMD extension name: `SpeciesType`
+
+
+Introduction
+------------
+
+This convention is for standardizing the names of particle species, e.g. in
+particle physics.
+
+
+Additional Record Attribute
+---------------------------
+
+The following additional attribute for openPMD `records` is defined in this
```

Comment:
oh wait, I think it should read:

... *mesh `records`* and *particle* `groups` ...

Reply:
How about simply:

The following attribute for openPMD is defined ...

Reply:
Sure, but I think for reader tools it is good to know where to look for it, no?

Reply:
OK I did not realize that this was defining where the info should go. How about adding a definition for `mesh record` and `particle group` in the Standard?

Reply:
good, this aspect is moved to #53